src/test/java/org/kainos/ea/serviceTests/JobRoleServiceTest.java

```diff
@@ -44,10 +42,17 @@ void getAllJobRoles_shouldThrowFailedToGetJobRoleException_whenDatabaseConnector
 
     @Test
     void getAllJobRoles_shouldReturnJobRoles_whenDaoReturnsJobRoles() throws DatabaseConnectionException, SQLException, FailedToGetJobRoleException {
-        List<JobRole> jobRole = new ArrayList<JobRole>();
+        List<JobRoleResponse> jobRoleResponseList = new ArrayList<>();
         when(databaseConnector.getConnection()).thenReturn(conn);
-        when(jobRoleDao.getAllJobRoles(conn)).thenReturn(jobRole);
-        List<JobRole> result = jobRoleService.getAllJobRoles();
-        assertEquals(jobRole, result);
+        when(jobRoleDao.getAllJobRoles(conn)).thenReturn(jobRoleResponseList);
+        List<JobRoleResponse> result = jobRoleService.getAllJobRoles();
+        assertEquals(jobRoleResponseList, result);
+
+        // Test capability is displayed
+        JobRoleResponse jobRoleResponse = new JobRoleResponse(1, "Technology Leader", "Engineering");
+        result.add(jobRoleResponse);
+        String expected = "Engineering";
+        String actual = result.get(0).getCapability();
+        assertEquals(expected, actual);
```

Comment:
The coverage from the controller test above might be enough. Not sure that these assertions are adding much value